### DISCUSSION TITLE
Translate between `Key` and `PhysicalKey`

### DISCUSSION THREAD

**Discussion by junglie85** [rust-windowing/winit#3182] [Q&A] [answered]
If I have a `Key<&str>`, is it possible to translate that to the `PhysicalKey` that was pressed? Equally, can we achieve the reverse given the `PhysicalKey`?

**Comment by kchibisov** [comment, marked as answer]
Not now, but maybe one day. It requires a keymap and translation is based on the keymap. You can not just match `"A"` to `KeyA`, without a keymap, I'm not even sure the reverse is possible on anything other than Linux.
